bin/worker_healthcheck.sh

```diff
@@ -0,0 +1,27 @@
+# rq info command returns monitoring status of redash workers and other status checks
+# Sample Output
+# redash@redash-adhocworker-778fc5dbb6-jpscq:/app$ rq info --url $REDASH_REDIS_URL -R
+# queries      | 0
+# default      | 2
+# periodic     |██████████████████ 748
+# emails       | 0
+# schemas      | 0
+# 5 queues, 750 jobs total
+
+# queries:  1a6d953d53984424ab775d3647427e7b (idle), 29dbf1ec1f774bacbd91ca36ccef6e43 (idle)
+# default:  –
+# periodic: –
+# emails:   –
+# schemas:  –
+# 2 workers, 5 queues
+
+WORKERS_COUNT=${WORKERS_COUNT}
```

Comment:
Thank you for adding this! Is there a reason we need the separate script?

Reply:
Yes we don't need it. Earlier I thought we would use this separate script in helm chart's liveness check.
But I realised there used to be healthcheck in docker entrypoint script.
I will remove this script.